source/templates.tex

```diff
@@ -3458,18 +3439,18 @@
 \end{example}
 
 \pnum
-In a type name that refers to a class template specialization, (e.g.,
-\tcode{A<int, int, 1>})
+In a name that refers to a specialization of a class or variable template,
+(e.g., \tcode{A<int, int, 1>})
```

Comment:
Please move the comma to after the "(e.g. ... )"

Reply:
fixed